src/components/data/viewers/FileViewer.js

```diff
@@ -45,6 +44,10 @@ import {
     Typography,
 } from "@material-ui/core";
 
+// these are at the bottom so eslint doesn't complain
+const TextViewer = dynamic(() => import("./TextViewer"));
+const VideoViewer = dynamic(() => import("./VideoViewer"));
```

Comment:
I am curious if we could use the async build provided `react-syntax-highlighter` directly and still get the same performance improvement. https://github.com/react-syntax-highlighter/react-syntax-highlighter#async-build

Reply:
I admit I couldn't get it to work, but I could give it another shot. We could get additional benefits from it, at least, because with the async build we could make sure we only load the languages that we actually use. If I did that, what I might consider doing is removing the dynamic imports here, but dynamically importing `FileViewer` itself (along with `react-syntax-highlighter`, of course). That way, the file viewer code doesn't end up loaded when just viewing listings, but all the viewers get loaded at once (and then the syntax highlighter only when needed).